dependencies.rosinstall

```diff
@@ -14,3 +14,7 @@
     local-name: opw_kinematics
     uri: https://github.com/Jmeyer1292/opw_kinematics.git
     version: master
+- git:
```

Comment:
I believe this can be removed. It should be included from the trajopt rosinstall.

Reply:
Are we merging them? I didn't think wstool could be run recursively, hence descartes_light in trajopt's rosinstall.